.github/workflows/ci.yml

```diff
@@ -13,7 +13,13 @@ jobs:
 
     strategy:
       matrix:
-        python-version: ["3.8", "3.9", "3.10", "3.11-dev"]
+        python-version:
+        # Build on pre-releases until stable, then stable releases.
```

Comment:
I think this comment could be clearer.
IIUC this syntax is a workaround that will pick the latest available version (which is what we want), and the version could be an alpha, a beta, or a final/stable release.

Reply:
I copied this comment verbatim from jaraco/skeleton. I agree, the comment could provide a whole narrative about why it exists, but I try to be concise, to describe as briefly as possible the reason for the syntax, and then link to an issue that provides more context and tracks ultimate resolution.

If you'd like to suggest different wording, I'd be willing to apply that here and in jaraco/skeleton.